### ISSUE TITLE
Feature Request: Add Support for MacOSX

### ISSUE THREAD

**Issue by jrock2004** [ronniedroid/getnf#4]
Would be awesome of script could be smart enough to know where to install based on if on Mac vs Linux. On Mac, fonts are installed to `$HOME/Library/Fonts`

I will gladly help try to implement if you like? But before I do the work, want to make sure that it would be something you would like

**Comment by jrock2004** [comment]
There seems to be an issue with M1 macs

**Comment by DaronAdams** [comment]
M1 mac here also having issues

**Comment by ronniedroid** [comment]
Can you please let me know what the problem is? An output maybe?

**Comment by jrock2004** [comment]
![image](https://user-images.githubusercontent.com/655716/198469488-47944f98-3f72-47d9-ac3a-167ecde2213f.png)

@ronniedroid here is the error I see

**Comment by shriharip** [comment]
on Mac 12 (Monterey), intel same issue.
```
Fonts Directory exists, good
Download Fonts Directory exists, good
grep: invalid option -- P
usage: grep [-abcdDEFGHhIiJLlMmnOopqRSsUVvwXxZz] [-A num] [-B num] [-C[num]]
	[-e pattern] [-f file] [--binary-files=value] [--color=when]
	[--context[=num]] [--directories=action] [--label] [--line-buffered]
	[--null] [pattern] [file ...]
Iosevka font download started
#################################################################################### 100.0%
Iosevka font download finished
Iosevka font unziping started
[Iosevka.zip]
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of Iosevka.zip or
        Iosevka.zip.zip, and cannot find Iosevka.zip.ZIP, period.
All is done!
```

**Comment by ronniedroid** [comment]
Thank you for the input, fixed with #9
